transcom/mymove
# INT. B-20109 refactor DocumentUploader to use Formik

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20109)

## Summary

We are not wanting to use `redux-form` and instead use Formik due to library support. This backlog item addresses the `DocumentUploader` component and its use of `redux-form` as well as accessory files.

Luckily, this component isn't being used ANYWHERE in the code base and was tech debt. Removing the file and placing it very gently six feet under should complete the task.

### How to test

1. In VSCode, search for `redux-form`
2. You should no longer see it being used in the `DocumentUploader` component
3. Confirm that `DocumentUploader` no longer exists in the code base and that it isn't being used anywhere else
